knesset_data/dataservice/bills.py

```diff
@@ -0,0 +1,39 @@
+from base import (
+    BaseKnessetDataServiceCollectionObject, BaseKnessetDataServiceFunctionObject,
+    KnessetDataServiceSimpleField, KnessetDataServiceLambdaField
+)
+import constants
+
+
+class Bill(BaseKnessetDataServiceCollectionObject):
+    SERVICE_NAME = "api"
+    METHOD_NAME = "KNS_Bill"
+    ORDERED_FIELDS = [
+        ("id", KnessetDataServiceSimpleField('BillID', "integer", "the primary key")),
+        ("kns_num", KnessetDataServiceSimpleField('KnessetNum', "integer", "kneset number")),
+        ("name", KnessetDataServiceSimpleField('Name', "string", "bill heb name")),
+        ("type_id", KnessetDataServiceSimpleField('SubTypeID', "integer", "type id of the bill")),
+        ("type_description",
+         KnessetDataServiceSimpleField('SubTypeDesc', "string", "type description of the bill")),
+        ("private_num", KnessetDataServiceSimpleField('PrivateNumber', "integer",)),
+        ("committee_id", KnessetDataServiceSimpleField('CommitteeID', "integer",)),
```

Comment:
this id field is related to Committee object
for this case you can add a helper method on your object which will try to fetch the related committe
something like this:

def get_committee(self):
  return Committee.get(self.committee_id)